app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/Intro.kt

```diff
@@ -0,0 +1,7 @@
+package com.codingblocks.cbonlineapp.auth.onboarding
+
+data class Intro(
```

Comment:
@aggarwalpulkit596 check if this is in the right place

Reply:
Could you add KDoc as well ? i was thinking to start adding Kdoc everywhere